scene/main/node.compat.inc

```diff
@@ -34,8 +34,13 @@ void Node::_set_name_bind_compat_76560(const String &p_name) {
 	set_name(p_name);
 }
 
+const Variant Node::_get_rpc_config_bind_compat_106848() {
```

Comment:
Shouldn't it be this to match the previous prototype?
```suggestion
Variant Node::_get_rpc_config_bind_compat_106848() const {
```

Reply:
Yep, fixed, thanks!